test/platform/target/network/scan_result_test.exs

```diff
@@ -0,0 +1,38 @@
+defmodule Farmbot.Target.Network.ScanResultTest do
```

Comment:
We should namespace this differently. Lets move the data struct `Farmbot.Target.Network.ScanResult` to a shared namespace `Farmbot.System.ScanResult` in `lib/farmbot/system/scan_result.ex` and add the `decode` functions there.